src/libraries/System.Data.OleDb/src/SafeHandles.cs

```diff
@@ -79,7 +79,7 @@ protected override bool ReleaseHandle()
 
     internal sealed class RowHandleBuffer : DbBuffer
     {
-        internal RowHandleBuffer(IntPtr rowHandleFetchCount) : base((int)rowHandleFetchCount * ADP.PtrSize)
+        internal RowHandleBuffer(nint rowHandleFetchCount) : base((int)rowHandleFetchCount * ADP.PtrSize)
```

Comment:
It would be nice find&replace `ADP.PtrSize` with `IntPtr.Size`, delete `ADP.PtrSize`.